### ISSUE TITLE
could you provide an example to execute a lambda

### ISSUE THREAD

**Issue by matth-c3** [Kong/lua-resty-aws#20]
Hello, 

could you provide an example on how to execute a lambda and return the result of this one ?

thanks

**Comment by Tieske** [comment]
This requires latest `main` branch. Fixes for it will land in `0.5.2` most likely

```lua
local AWS = require("resty.aws")
local aws = AWS()


local myLambda = aws:Lambda { region = "us-east-2" }
local ok, err = myLambda:invoke {
  FunctionName = "testFunction",
  Payload = '{ "John": "Smith" }',
}
```

**Comment by Tieske** [comment]
just released `0.5.2` 🎉
